From 46a55ed8f7ef4ee99ebccf1bdded3d5ccaef33af Mon Sep 17 00:00:00 2001
From: Jiachen YANG <farseerfc@gmail.com>
Date: Fri, 4 Oct 2019 17:00:48 +0900
Subject: [PATCH 5/6] Revert "Change some private header includes."

This reverts commit b9d3ba621eb8af638af46c6b3cfd7a8330bf0dd5.
---
 Telegram/SourceFiles/ui/text/text.cpp       | 1 -
 Telegram/SourceFiles/ui/text/text.h         | 2 +-
 Telegram/SourceFiles/ui/text/text_block.cpp | 2 --
 Telegram/SourceFiles/ui/text/text_block.h   | 1 +
 4 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/Telegram/SourceFiles/ui/text/text.cpp b/Telegram/SourceFiles/ui/text/text.cpp
index eaeb52b35..2c55f1123 100644
--- a/Telegram/SourceFiles/ui/text/text.cpp
+++ b/Telegram/SourceFiles/ui/text/text.cpp
@@ -14,7 +14,6 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 #include "ui/ui_integration.h"
 #include "platform/platform_info.h"
 
-#include <private/qfontengine_p.h>
 #include <private/qharfbuzz_p.h>
 
 namespace Ui {
diff --git a/Telegram/SourceFiles/ui/text/text.h b/Telegram/SourceFiles/ui/text/text.h
index 5d7c2cd07..3a2de5d3e 100644
--- a/Telegram/SourceFiles/ui/text/text.h
+++ b/Telegram/SourceFiles/ui/text/text.h
@@ -12,7 +12,7 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 #include "ui/click_handler.h"
 #include "base/flags.h"
 
-#include <private/qfixed_p.h>
+#include <private/qfontengine_p.h>
 
 static const QChar TextCommand(0x0010);
 enum TextCommands {
diff --git a/Telegram/SourceFiles/ui/text/text_block.cpp b/Telegram/SourceFiles/ui/text/text_block.cpp
index 8e6914197..d83073085 100644
--- a/Telegram/SourceFiles/ui/text/text_block.cpp
+++ b/Telegram/SourceFiles/ui/text/text_block.cpp
@@ -9,8 +9,6 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 
 #include "styles/style_basic.h"
 
-#include <private/qfontengine_p.h>
-
 // COPIED FROM qtextlayout.cpp AND MODIFIED
 namespace Ui {
 namespace Text {
diff --git a/Telegram/SourceFiles/ui/text/text_block.h b/Telegram/SourceFiles/ui/text/text_block.h
index 6fcf1cd51..27d724ee8 100644
--- a/Telegram/SourceFiles/ui/text/text_block.h
+++ b/Telegram/SourceFiles/ui/text/text_block.h
@@ -11,6 +11,7 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 #include "ui/emoji_config.h"
 
 #include <private/qfixed_p.h>
+#include <private/qfontengine_p.h>
 
 namespace Ui {
 namespace Text {
-- 
2.23.0

